packages/widget-chat/src/containers/chat-widget/index.js

```diff
@@ -83,24 +84,6 @@ export class ChatWidget extends Component {
     }
   }
 
-  /* eslint-disable complexity */
-  /* eslint-disable-reason: Lots of checks for better efficiency, will be broken up later */
-  shouldComponentUpdate(nextProps) {
-    const props = this.props;
-
-    /* eslint-disable operator-linebreak */
-    /* eslint-disable-reason: Giant list of comparisons very difficult to read and diff */
-    return nextProps.conversation.activities !== props.conversation.activities
-      || nextProps.conversation.isLoadingHistoryUp !== props.conversation.isLoadingHistoryUp
-      || nextProps.flags !== props.flags
-      || nextProps.indicators !== props.indicators
-      || nextProps.share !== props.share
-      || nextProps.sparkState.connected !== props.sparkState.connected
```

Comment:
If we remove this check, the `isLoadingHistoryUp`, and the `activities` check, does this mean that any time the `sparkState` and `conversation` stores change we'll re-render the widget? I think that may cause quite a few extra re-renders, but not 100% sure. The `react-redux` docs seem to suggest that it only does shallow comparisons.